indexer-app/src/cmr/indexer/data/concepts/collection.clj

```diff
@@ -388,6 +388,8 @@
             :metadata-format (name (mt/format-key format))
             :related-urls (map json/generate-string opendata-related-urls)
             :has-opendap-url (not (empty? (filter opendap-util/opendap-url? related-urls)))
+            :cloud-hosted (or (not (empty? (:DirectDistributionInformation collection)))
+                               (tag/has-cloud-s3-tag tags))
```

Comment:
The space doesn't seem to line up. It looks like you have 1 extra space by (tag...) But the parens are ok.

Reply:
moved to a private function, did alignment.